packages/compiler_utils/src/dependencies.candy

```diff
@@ -0,0 +1,81 @@
+use incremental
+use petit_parser
+use ..error
+use ..file
+use ..location
+use ..package
+
+impl Package {
+  public fun getAllDependencies(context: QueryContext<List<CompilerError>>): Set<Package> {
```

Comment:
Can't we reduce `getAllDependencies` to `getAllPackagesInCompilationOrder`? As in, couldn't we implement `getAllDependencies` like this:
```kotlin
public fun getAllDependencies(context: QueryContext<List<CompilerError>>): Set<Package> {
  getAllPackagesInCompilationOrder(context).toSet()
}
```
Then, cycle checking would also need to be included in `getAllPackagesInCompilationOrder`.